pkg/pfconfig/lbslns/lbslns.go

```diff
@@ -231,7 +234,7 @@ func (conf RouterConfig) MarshalJSON() ([]byte, error) {
 
 // GetRouterConfig returns the routerconfig message to be sent to the gateway.
 // Currently as per the basic station docs, all frequency plans have to be from the same region (band) https://doc.sm.tc/station/tcproto.html#router-config-message.
-func GetRouterConfig(bandID string, fps map[string]*frequencyplans.FrequencyPlan, isProd bool, dlTime time.Time) (RouterConfig, error) {
+func GetRouterConfig(bandID string, fps map[string]*frequencyplans.FrequencyPlan, isProd bool, dlTime time.Time, antennaGain int) (RouterConfig, error) {
```

Comment:
This is getting quite crowded, we should consider a `struct` / `Options` in the future.